API/.env

```
@@ -0,0 +1,4 @@
+DJANGO_SECRET_KEY='django-insecure-*ppblt%cgqvj82s(p-mqk0s&ib^#51pr64b(g!%%)i#$n3z=_x'
+DJANGO_DEBUG=True
+DJANGO_ALLOWED_HOSTS=localhost,127.0.0.1
+DJANGO_ENV=development
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)